zebrad/src/components/mempool.rs

```diff
@@ -141,6 +146,24 @@ impl Service<Request> for Mempool {
                 let _ = self.storage.insert(tx);
             }
         }
+
+        let mut storage = self.storage.clone();
+        let mut state = self.state.clone();
+
+        let _ = tokio::task::spawn(async move {
```

Comment:
If you decide to switch to `LatestChainTip::best_tip_height`, you won't need to spawn a task, because `LatestChainTip` stores the latest tip height every time the state changes.

Reply:
Sorry, i missed this one, will remove the task spawn.